### PR TITLE
Abyssea Aggro and Linking Fixes

### PR DESCRIPTION
**Pull request by R3P0FFXI**
Fixes for all 9 Abyssea zones

### CODE REVIEW

File: sql/mob_pools.sql [hunk 1]
```diff
@@ -1998,11 +1998,11 @@ INSERT INTO `mob_pools` VALUES (1937,'Hetairoi_Vuryargur','Hetairoi_Vuryargur',2
...
 INSERT INTO `mob_pools` VALUES (1939,'Hexenpilz','Hexenpilz',116,0x0000C90800000000000000000000000000000000,1,8,6,240,100,0,0,0,1,2,0,0,0,0,0,0,0,1,0,800);
 INSERT INTO `mob_pools` VALUES (1940,'He_Dho_Spatesurge','He_Dho_Spatesurge',200,0x00009A0200000000000000000000000000000000,1,1,0,0,100,0,0,0,0,0,0,0,0,0,0,0,0,1,0,200);
-INSERT INTO `mob_pools` VALUES (1941,'Hieracosphinx','Hieracosphinx',179,0x0000920100000000000000000000000000000000,1,1,7,360,100,0,1,1,1,16,0,0,6,135,0,0,0,1,0,179);
-INSERT INTO `mob_pools` VALUES (1942,'Highland_Rafflesia','Highland_Rafflesia',207,0x0000E80700000000000000000000000000000000,9,8,7,240,100,0,0,0,0,0,0,0,0,0,0,0,0,1,0,207);
-INSERT INTO `mob_pools` VALUES (1943,'Highland_Treant','Highland_Treant',245,0x0000840100000000000000000000000000000000,1,1,8,240,100,0,0,0,0,0,0,0,1,133,0,0,0,1,0,245);
+INSERT INTO `mob_pools` VALUES (1941,'Hieracosphinx','Hieracosphinx',179,0x0000920100000000000000000000000000000000,1,1,7,360,100,0,1,0,0,16,0,0,6,135,0,0,0,1,0,179);
+INSERT INTO `mob_pools` VALUES (1942,'Highland_Rafflesia','Highland_Rafflesia',207,0x0000E80700000000000000000000000000000000,9,8,7,240,100,0,1,0,1F,0,0,0,0,0,0,0,0,1,0,207);
```
Comment (TeoTwawki) [inline sql/mob_pools.sql]:
```
Importing mob_pools.sql into the database...ERROR 1054 (42S22) at line 2002: Unknown column '1F' in 'field list'
```

Comment (R3P0FFXI) [inline sql/mob_pools.sql]: whoops fucked up a control + F

File: sql/mob_pools.sql [hunk 2]
```diff
@@ -124,7 +124,7 @@ INSERT INTO `mob_pools` VALUES (63,'Agonizer','Agonizer',241,0x00006804000000000
 INSERT INTO `mob_pools` VALUES (64,'Agrios','Agrios',128,0x0000120100000000000000000000000000000000,1,1,11,240,100,0,1,1,0,2,0,32,153,1181,0,0,0,1,0,128);
 INSERT INTO `mob_pools` VALUES (65,'Ahriman','Ahriman',4,0x0000080100000000000000000000000000000000,1,4,7,240,100,0,1,0,1,1,0,0,407,1157,0,0,11,1,0,4);
 INSERT INTO `mob_pools` VALUES (66,'Ahtu','Ahtu',245,0x0000840100000000000000000000000000000000,1,1,7,240,100,0,1,0,0,2,0,0,711,1157,0,0,0,1,0,245);
-INSERT INTO `mob_pools` VALUES (67,'Aht_Urhgan_Attercop','Aht_Urhgan_Attercop',235,0x0000380100000000000000000000000000000000,1,1,7,240,100,0,0,0,1,0,0,0,1,131,0,0,0,1,0,235);
+INSERT INTO `mob_pools` VALUES (67,'Aht_Urhgan_Attercop','Aht_Urhgan_Attercop',235,0x0000380100000000000000000000000000000000,1,1,7,240,100,512,1,0,1,0,0,0,1,131,0,0,0,1,0,235);
```
Comment (TeoTwawki) [inline sql/mob_pools.sql]: This doesn't belong on *Aht Urhgan Attercop*'s
```cpp
BEHAVIOUR_AGGRO_AMBUSH = 512, // mob aggroes by ambush
```

Comment (TeoTwawki) [inline sql/mob_pools.sql]: also they don't agro, just link

**Comment by TeoTwawki** [comment]
does turdhub let you reopen? I want to try and fix this. Next time don't be so quick to hammer close when you mess something up between your local copy and the remote.

**Pull request merged**